### DISCUSSION TITLE
Nested layout (pathless route) not working

### DISCUSSION THREAD

**Discussion by johnx888** [TanStack/router#2702] [Q&A] [answered]
In react router, I can do something like this:
```js
{
  path: '',
  element: <AuthenticatedRoutes/>,
  children: [
    {
      path: '',
      element: <MainLayout/>,
      children: [
        {
          path: '/a',
          element: <A />
        },
        {
          path: '/b',
          element: <B />
        }
      ]
    },
    {
      path: '',
      element: <AnotherLayout/>,
      children: [
        {
          path: '/c',
          element: <C />
        },
        {
          path: '/d',
          element: <D />
        }
      ]
    }
  ]
}
```
I try to create something similar with file-based routing
```
-------- __root.tsx
-------- /_authenticated
---------------- layout.tsx
---------------- thisWillWork.lazy.tsx
---------------- /_mainLayout
-------------------------------- layout.tsx
-------------------------------- a.lazy.tsx 
-------------------------------- b.lazy.tsx
```
But It return empty white page when I'm navigate to /a and /b and /thisWillWork still work.

Is there any way to achieve this with current version of tanstack router? If no, is there any workaround for this?

Thanks a lot.

**Comment by schiller-manuel** [comment]
can you please provide a complete minimal example, e.g. by forking one of the existing router examples (e.g. https://tanstack.com/router/v1/docs/framework/react/examples/basic-file-based) ?

**Reply by johnx888** [reply, marked as answer]
Sorry for bothering you guys. It turned out to be my silly mistake; I accidentally deleted the return statement in my MainLayout file. I realize this when forking that example and reproducing it.
